docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -164,12 +164,7 @@ options are `ssh` and `k8s`.
 
 A participant joining a session will always have one of three modes:
 
-- `peer`: Can join and collaborate in a session. They can view output and send
-  input.
-- `moderator`: Can join and watch a session. They can view output and
-  forcefully terminate the session at will.
-- `observer`: Can join and watch a session. They cannot control the session in
-  any way.
+(!docs/pages/includes/participant-modes.mdx!)
```

Comment:
Are the amount and/or definition of the roles likely to change/expand in the future? Are we likely to distribute this list to more pages than the two in this PR?

If the answers to those questions is 'no', then I don't think we get more value out of a partial file than we get hassle in managing it.